app/templates/response/edit_user_request.js.html

```diff
@@ -64,7 +64,9 @@
 
         next.click(function () {
             // check if permissions are different
-            if (!permission.val()
+            if (!(permission.val() === null && current_permissions.length === 0) ||
+                permission.val() === null ||
```

Comment:
I guess we rushed. Should be:
``` javascript
(permission.val() === null && current_permissions.length > 0)
```
Just checking if permissions are null will void our first check.